content/pages/conference/index.rst

```diff
@@ -0,0 +1,18 @@
+===========
+Conference
+===========
+
+:date: 2023-09-28 09:00
+:url: conference
+:save_as: conference/index.html
+
+`conference 2023 <https://conf.sciwork.dev>`__
```

Comment:
It's better to put the link in the body in the section.  It is not obvious that the section title takes you to a new website.